utility/src/test/java/org/oppia/android/util/profile/ProfileNameValidatorTest.kt

```diff
@@ -0,0 +1,71 @@
+package org.oppia.android.util.profile
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.Before
+import org.junit.Test
+import javax.inject.Inject
+
+class ProfileNameValidatorTest {
+  @Inject
+  lateinit var profileNameValidator: ProfileNameValidator
+
+  @Before
+  fun setup() {
+    profileNameValidator = ProfileNameValidator()
```

Comment:
This should be injected as I mentioned earlier. Are you running into any issues with setting this up as a Robolectric test? I suggest referencing other utility and domain tests for a reference (you probably won't need any modules nor a custom component, so it should be a relatively simple setup).

Reply:
done :)